cluster-scope/components/project-admin-rolebindings/peribolos/rbac.yaml

```diff
@@ -0,0 +1,12 @@
+apiVersion: rbac.authorization.k8s.io/v1
+kind: RoleBinding
+metadata:
+    name: namespace-admin-peribolos
+roleRef:
+    apiGroup: rbac.authorization.k8s.io
+    kind: ClusterRole
+    name: admin
+subjects:
+    - apiGroup: rbac.authorization.k8s.io
+      kind: Group
+      name: peribolos
```

Comment:
group name here should match the one being created above `peribolos-as-a-service`

Reply:
@4n4nd should this dir name also be `peribolos-as-a-service`?